### PR TITLE
add hostname in upload notifications

### PR DESCRIPTION
**Pull request by atonks2**

### CODE REVIEW

**Comment by atonks2** [comment]
For issue #502

File: go.mod [hunk 1]
```diff
@@ -3,51 +3,70 @@ module github.com/moov-io/paygate
 go 1.13
 
 require (
```
Comment (adamdecaf) [inline go.mod:5]: Can we undo this change? Let's rebase this branch after https://github.com/moov-io/paygate/pull/597 is merged.

File: pkg/config/pipeline.go [hunk 2]
```diff
@@ -15,7 +15,7 @@ import (
 
 var (
 	DefaultEmailTemplate = template.Must(template.New("email").Parse(`
-A file has been {{ .Verb }}ed from {{ .CompanyName }}: {{ .Filename }}
+A file has been {{ .Verb }}ed {{ if and (eq .Verb "upload") (.Hostname) }}to {{ .Hostname }} {{ end }}from {{ .CompanyName }}: {{ .Filename }}
```
Comment (adamdecaf) [inline pkg/config/pipeline.go:18]: Why only on upload? I think `{{ if (.Hostname) }}` would be good enough. It's helpful to know if files fail to download as well.

File: pkg/upload/ftp.go [hunk 3]
```diff
@@ -186,6 +187,23 @@ func (agent *FTPTransferAgent) ReturnPath() string {
 	return agent.cfg.ReturnPath
 }
 
+func (agent *FTPTransferAgent) Hostname() string {
+	if agent.cfg.FTP == nil {
+		return ""
+	}
+
+	host := agent.cfg.FTP.Hostname
+	if !strings.Contains(host, ":") {
```
Comment (adamdecaf) [inline pkg/upload/ftp.go:196]: Is there a reason we'd drop the port? It might be helpful to include that - I didn't think about stripping the port in the original issue. 

I've found that on-call notifications should include exactly the config/error occurring. Modify a config value like this has caused me to dig into the wrong thing - what if the port really is missing or misconfigured?

File: pkg/config/pipeline.go (hunk 2 above)
Comment (atonks2) [reply pkg/config/pipeline.go:18]: I can switch it. I did upload only because of the description in the issue:
> When getting an email / slack message **on file upload** it's helpful to know where it was uploaded to. This could be the hostname for now.

File: pkg/upload/ftp.go (hunk 3 above)
Comment (atonks2) [reply pkg/upload/ftp.go:196]: I can include the port.

File: go.mod (hunk 1 above)
Comment (atonks2) [reply go.mod:5]: No problem.

**Comment by atonks2** [comment]
New Slack message:


New email template:
```
A file has been uploaded to ftp.bank.com:3294 - 20200529-131400.ach
Name: Moov
Debits:  $105.00
Credits: $0.00

Batches: 1
Total Entries: 1
```

File: pkg/config/pipeline.go (hunk 2 above)
Comment (adamdecaf) [reply pkg/config/pipeline.go]: Ahh okay. I think we should have it on both, so I probably was focused on upload errors at the time.

**Review by adamdecaf — APPROVED** [review]
This is awesome, thank you!
